recipe/build_opencv.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+set -ex
+
+# CMake FindPNG seems to look in libpng not libpng16
+# https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindPNG.cmake#L55
+ln -s $PREFIX/include/libpng16 $PREFIX/include/libpng
+
+V4L="1"
+
+if [[ "${target_platform}" == linux-* ]]; then
+    # Looks like there's a bug in Opencv 3.2.0 for building with FFMPEG
+    # with GCC opencv/issues/8097
+    export CXXFLAGS="$CXXFLAGS -D__STDC_CONSTANT_MACROS"
+    OPENMP="-DWITH_OPENMP=1"
```

Comment:
I'm not seeing openmp in the deps. Also is there any blas options to switch on?

Reply:
The openmp mutex is in the run dependencies for linux, and llvm-openmp for osx. Through the compiler.
No blas for opencv (except cublas, but we are not doing the cuda variant).

Reply:
the openmp mutex doesn't pull in openmp, and imho would be better to include the openmp packages explicitly. Also are we using intel-openmp for x86_64 platforms? being explicit would show that

Reply:
That's inline with how the recipe was before and looks alright to me. llvm-openmp is not used because WITH_OPENMP is not set on the osx variant. That's about the only change I would make.

Reply:
ok